src/validations/advancedTransactionValidations.ts

```diff
@@ -0,0 +1,27 @@
+import { ValidationFunction } from '../hooks/useErrorHandler';
+import { getParamFromUrlOrState } from '../utils/urlHelpers';
+
+const validateTransactionData: ValidationFunction = ({ urlParams, decodedState }) => {
+  const transactionData = getParamFromUrlOrState(
+    'transactionData',
+    urlParams,
+    decodedState,
+  );
+
+  if (!transactionData) {
+    return {
+      isValid: false,
+      error: {
+        title: 'Missing Transaction Data',
+        message:
+          'Missing transaction data, please check the configuration and reload the page.',
```

Comment:
Let's remove `the page` - it spills over to the second line on web and looks weird.